packages/form-builder/src/components/canvas/FormBlocks.tsx

```diff
@@ -2,13 +2,16 @@ import {BlockList, BlockTools, DefaultBlockAppender, ObserveTyping, WritingFlow}
 
 const FormBlocks = () => {
     return (
-        <BlockTools>
-            <WritingFlow>
-                <ObserveTyping>
-                    <BlockList renderAppender={DefaultBlockAppender} />
-                </ObserveTyping>
-            </WritingFlow>
-        </BlockTools>
+        <div id="FormBlocksCanvas" style={{padding: '58px', margin: '-58px'}}>
+            <BlockTools>
```

Comment:
These styles make me a little uneasy, what's this for?

Reply:
This is specifically to set spacing when highlighting the canvas. There is not actual spacing the space the canvas wrapper from the block list wrapper, so this inline style is used to set the location of the outline.

Reply:
Instead, we could target the id and conditionally apply this style when targeted by ShepherdJS.

Reply:
I migrated the inline styles to the stylesheet and set the selector to only apply when targeted by ShepherdJS, see https://github.com/impress-org/givewp-next-gen/pull/143/commits/4f5b7de247394ff709d04977590dddb46efb7869.

```css
#FormBlocksCanvas.shepherd-target {
    padding: 58px;
    margin: -58px;
}
```

Reply:
okay cool, what happens if you don't have these styles?

Reply:
The boarder collapses, which makes it difficult to distinguish between steps one and two.